hs_access_control/models.py

```diff
@@ -2285,6 +2426,29 @@ def get_resources_with_explicit_access(self, this_privilege):
 
             return BaseResource.objects.filter(Q(pk__in=view) | Q(pk__in=immutable)).distinct()
 
+    def get_users_with_explicit_access(self, this_privilege):
+        """
+            Get a list of users for which the group has the specified privilege
+
+            :param this_privilege: one of the PrivilegeCodes
+            :return: QuerySet of user objects (QuerySet)
+        """
+
+        if __debug__:
+            assert this_privilege >= PrivilegeCodes.OWNER and this_privilege <= PrivilegeCodes.VIEW
+
+        if this_privilege == PrivilegeCodes.OWNER:
+            return self.owners
+        elif this_privilege == PrivilegeCodes.CHANGE:
+            return User.objects.filter(is_active=True,
+                                       u2ugp__group=self.group,
+                                       u2ugp__privilege=PrivilegeCodes.CHANGE)
+
+        else:  # this_privilege == PrivilegeCodes.ViEW
+            return User.objects.filter(is_active=True,
+                                       u2ugp__group=self.group,
+                                       u2ugp__privilege=PrivilegeCodes.VIEW)
+
```

Comment:
@pkdash rather than using assert to make sure this_privilege parameter is valid, how about change line 2447 to ```elif this_privilege == PrivilegeCodes.VIEW``` and add the last ```else:``` to return None if this_privilege is not valid?